samples/client/petstore/typescript-rxjs/builds/default/runtime.ts

```diff
@@ -47,12 +47,18 @@ export class Configuration {
 
     get apiKey(): ((name: string) => string) | undefined {
         const apiKey = this.configuration.apiKey;
-        return apiKey && (typeof apiKey === 'function' ? apiKey : () => apiKey);
+        if (!apiKey) {
+            return undefined;
+        }
+        return typeof apiKey === 'string' ? () => apiKey : apiKey;
```

Comment:
I'm having an issue with the petstore script. @amakhrov maybe you can help? 

No matter how many times / ways I run it, it generates code with an `if` conditional instead of a ternary, referencing an older version of the template (I did a search in the entire repo and these lines of code are only found in the generated code).

Reply:
Never mind. I think I found the issue.